packages/cli/src/commands/bundle/buildBundle.ts

```diff
@@ -50,6 +50,9 @@ async function buildBundle(
     config: args.config,
   });
 
+  // Don't need file watcher for building offline bundle
+  config.watch = false;
```

Comment:
Please don't mutate the config object like this. Also, isn't this failing validation built into `metro-config`? We'll likely need to wait for a new version and include it after Metro upgrade

Reply:
Thanks, I'll update that.

Regarding validation - I didn't see any validation errors in my local workspace with the corresponding changes to metro.

I updated the description to make it clear that this PR depends on a release of https://github.com/facebook/metro/pull/491 before it can land.